crypto/sha/asm/sha256-mb-x86_64.pl

```diff
@@ -542,23 +542,23 @@ sub ROUND_16_XX {
 	movdqa		0*16-0x80($Tbl),$Wi
 	pshufb		$TMPx,@MSG0[1]
 	paddd		@MSG0[0],$Wi
-	pxor		$ABEF0,@MSG0[0]		# black magic
```

Comment:
I wonder if it's really offensive to anybody, or is it just guessing. I don't see any relations with races here at all. I'm not a native English speaker, but to me, it seems even more inappropriate to call a race "black" (because technically their skin color is dark brown, not black) than to ban every phrase that has the word "black" in it. We might end up banning "black" as a color or renaming "black hole" into "hole" (I doubt it would make e.g. Neil Degrass Tyson happier though).